coriolis/api/v1/deployments.py

```diff
@@ -80,9 +81,9 @@ def _validate_deployment_input(self, context, body):
             'instance_osmorphing_minion_pool_mappings', {})
         user_scripts = deployment.get('user_scripts', {})
         api_utils.validate_user_scripts(user_scripts)
-        user_scripts = api_utils.normalize_user_scripts(
-            user_scripts, deployment.get("instances", []))
-
+        import remote_pdb; remote_pdb.RemotePdb('10.8.254.30', 4444).set_trace()
+        user_scripts = ConductorServerEndpoint._normalize_user_scripts(
```

Comment:
The reason for this fix was to only run this normalization in the conductor itself, not make the API layer run it from the Conductor layer. (that's why I asked for it to be moved into the conductor)
Please remove any kind of user script normalization from the API layer.